spec/models/form/sales/questions/owning_organisation_id_spec.rb

```diff
@@ -206,20 +209,21 @@
 
     context "when user is support" do
       let(:user) { create(:user, :support, organisation: organisation_1) }
-
       let(:log) { create(:lettings_log, created_by: user) }
 
-      let(:non_stock_organisation) { create(:organisation, holds_own_stock: false) }
```

Comment:
The timing here meant that this didn't organisation didn't actually exist at the relevant point during the tests.